src/PlaywrightSharp.Tests/BaseTests/PlaywrightSharpLoaderFixture.cs

```diff
@@ -7,27 +7,17 @@ namespace PlaywrightSharp.Tests.BaseTests
     /// <summary>
     /// This class will build all http servers and download browsers
     /// </summary>
-    public class PlaywrightSharpLoaderFixture : IDisposable
+    public class PlaywrightSharpLoaderFixture
     {
         internal static SimpleServer Server { get; private set; }
         internal static SimpleServer HttpsServer { get; private set; }
 
         /// <inheritdoc/>
-        public PlaywrightSharpLoaderFixture()
-        {
-            SetupAsync().GetAwaiter().GetResult();
-        }
-
-        /// <inheritdoc />
-        public void Dispose()
-        {
-            Task.WaitAll(Server.StopAsync(), HttpsServer.StopAsync());
-        }
+        static PlaywrightSharpLoaderFixture() => SetupAsync().GetAwaiter().GetResult();

```

Comment:
had issues where multiple initializers were calling this and throwing due to the server running already.